templates/etc/td-agent/td-agent.conf

```diff
@@ -20,6 +20,12 @@
   @type null
 </match>
 
+# Add a unique insertId to each log entry that doesn't already have it. This
```

Comment:
[Optional] At this point, I'd move the `This` to the next line to keep the sentence together. Ditto below.

Reply:
Done